src/foam/u2/crunch/lab/CapabilityGraphNodeView.js

```diff
@@ -102,17 +103,21 @@ foam.CLASS({
             .addClass(this.myClass('segment')).addClass('title')
             .add(foam.String.labelize(capability.name))
           .end()
-          .start()
+          .start('button')
             .addClass(this.myClass('segment')).addClass('tiny')
             .add(capability.id)
+            .on('click', function() { 
+              window.location.href = `http://${location.host}/#admin.data:capabilityDAO:{sV:%22New%20Controller%22,c:[%22id%22,%22name%22,%22description%22]}:${capability.id}:{sV:%22Detail%22,c:[%22granted%22,%22id%22,%22description%22]}`;
```

Comment:
Does this escaping work correctly?

Reply:
https://user-images.githubusercontent.com/10802216/104962162-d8ff0680-59a5-11eb-9870-756d31e075d8.mov


https://user-images.githubusercontent.com/10802216/104962200-ecaa6d00-59a5-11eb-967c-a0b599a4499f.mov

Reply:
Better to 
ctrl.__subContext__.memento.value = "admin.data:capabilityDAO:...";
Or, in application controller we should add 'memento as topMemento' to exports,then import that here and do:
this.topMemento.value = "admin.data:capabilityDAO:...";

Reply:
I didn't see any issues

Reply:
One issue I have with this is that if @olhabn changes the memento signature then it will silently break this code.